config/environments/shared_deployment_config.rb

```diff
@@ -31,7 +31,7 @@
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
-
+  
```

Comment:
Irrelevant whitespace change; truly not a blocker, but I saw it, so there you go.

Reply:
i edited this line and now my editor is being stubborn about adding this tab on the empty line... tried for 10 seconds to remove it and it didnt work so im giving up :D